rt/rs/microprofile-client/src/test/java/org/apache/cxf/microprofile/client/CxfTypeSafeClientBuilderTest.java

```diff
@@ -103,41 +108,28 @@ public void testInvokesPostOperationWithRegisteredProviders() throws Exception {
         //assertEquals(TestWriterInterceptor.getAndResetValue(), 1);
     }
 
-/** using for test coverage
-    @Override
-    public RestClientBuilder register(Class<?> componentClass, int priority) {
-      configImpl.register(componentClass, priority);
-      return this;
+    @Test(expected = NoSuchEntityException.class)
+    public void testResponseExceptionMapper() throws Exception {
+        ExceptionMappingClient client = new CxfTypeSafeClientBuilder()
+            .register(NotFoundExceptionMapper.class)
+            .register(NotFoundClientReqFilter.class)
+            .baseUrl(new URL("http://localhost/null"))
+            .build(ExceptionMappingClient.class);
+
+        Response r = client.getEntity();
+        System.out.println(r.getStatus());
```

Comment:
We can probably move the SOP into the `fail` method.

Reply:
Agreed.  Will move.  Thanks for the review!